apps/web/pages/index.tsx

```diff
@@ -255,7 +255,7 @@ const Home: NextPage = () => {
                     <span>useSubscribe</span>
                     <Button
                       leftIcon={<FaBell />}
-                      onClick={subscribe}
+                      onClick={() => subscribe()}
```

Comment:
This is updated I guess it's because throwing type error. Shouldn't it actually be okay with a function that doesn't have params?  I mean it should be happy with it

Reply:
No it's more than that, if passed as `subscribe` to `onclick`, `onclick` will pass an `Event` to `susbcribe` as an `account` since `subscribe` now  accepts optional parameters